addons/website_quote_layout/views/website_quote_layout.xml

```diff
@@ -0,0 +1,93 @@
+                                      t-att-style="line.discount and 'text-decoration: line-through' or ''"
+                                      t-att-class="line.discount and 'text-danger' or ''"/>
+                                    <div t-if="line.discount">
+                                        <t t-esc="'%.2f' % ((1-line.discount / 100.0) * line.price_unit)"/>
+                                    </div>
+                                </strong>
+                            </td>
+                            <td class="o_website_layout_nowrap">
+                                <div class="text-right"
+                                  t-field="line.price_subtotal"
+                                  t-field-options='{"widget": "monetary", "display_currency": "quotation.pricelist_id.currency_id"}'/>
+                            </td>
+                            <td  t-if="line.option_line_id">
+                                <a t-attf-href="./update_line/#{ line.id }/?order_id=#{ quotation.id }&amp;unlink=True&amp;token=#{ quotation.access_token }" class="mb8 js_update_line_json pull-right hidden-print">
+                                    <span class="fa fa-trash-o"></span>
+                                </a>
+                            </td>
+                        </tr>
+                    </t>
+                    <t t-call="website_quote_layout.subtotal_template" />
+                        <tr class="text-center">
+                            <td colspan="6">
+                                ***
+                            </td>
+                        </tr>
+                    </t>
+                    <tr>
+                        <td class="text-right" colspan="5"><strong>Subtotal:</strong></td>
+                        <td class="text-right o_website_layout_nowrap">
+                            <strong data-id="total_amount" t-field="quotation.amount_untaxed" t-field-options='{"widget": "monetary","display_currency": "quotation.pricelist_id.currency_id"}'/>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="text-right" colspan="5">Taxes:</td>
+                        <td class="text-right o_website_layout_nowrap">
+                            <span data-id="total_amount" t-field="quotation.amount_tax" t-field-options='{"widget": "monetary","display_currency": "quotation.pricelist_id.currency_id"}'/>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="text-right" colspan="5"><strong>Total:</strong></td>
+                        <td class="text-right o_website_layout_nowrap">
+                            <strong data-id="total_amount" t-field="quotation.amount_total" t-field-options='{"widget": "monetary","display_currency": "quotation.pricelist_id.currency_id"}'/>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
```

Comment:
"page break" does not work :s 
if 'pagebreak' activate, split the table, like in sale_layout.